.github/workflows/nodejs.yml

```diff
@@ -21,3 +21,28 @@ jobs:
         run: npm run test
       - name: Build TypeScript
         run: npm run build
+  deploy:
+    if: github.event_name == 'push' && github.ref == 'refs/heads/master'
+    needs: test
+    runs-on: ubuntu-latest
+    env:
+      GKE_CLUSTER: ${{ secrets.GKE_CLUSTER }}
+      GCE_ZONE: ${{ secrets.GCE_ZONE }}
+      APP_ID: ${{ secrets.APP_ID }}
```

Comment:
These (APP_ID, PRIVATE_KEY, WEBHOOK_SECRET) should be set in environment variables of the Pod. You can update the Secret in the internal cluster to include these values.